File: add/codesnippet/CSharp/p-system.enterpriseservi_24_1.cs
[ObjectPooling(false)]
public class ObjectPoolingAttribute_Enabled : ServicedComponent
{
    public void EnabledExample()
    {
        // Get the ObjectPoolingAttribute applied to the class.
        ObjectPoolingAttribute attribute =
            (ObjectPoolingAttribute)Attribute.GetCustomAttribute(
            this.GetType(),
            typeof(ObjectPoolingAttribute),
            false);

        // Display the current value of the attribute's Enabled property.
        Console.WriteLine("ObjectPoolingAttribute.Enabled: {0}",
            attribute.Enabled);

        // Set the Enabled property value of the attribute.
        attribute.Enabled = true;

        // Display the new value of the attribute's Enabled property.
        Console.WriteLine("ObjectPoolingAttribute.Enabled: {0}",
            attribute.Enabled);
    }
}

File: add/codesnippet/CPP/p-system.enterpriseservi_24_1.cpp
[ObjectPooling(false)]
public ref class ObjectPoolingAttributeEnabled : public ServicedComponent
{
public:
    void EnabledExample()
    {
        // Get the ObjectPoolingAttribute applied to the class.
        ObjectPoolingAttribute^ attribute =
            (ObjectPoolingAttribute^)Attribute::GetCustomAttribute(
            this->GetType(),
            ObjectPoolingAttribute::typeid,
            false);

        // Display the current value of the attribute's Enabled property.
        Console::WriteLine("ObjectPoolingAttribute.Enabled: {0}",
            attribute->Enabled);

        // Set the Enabled property value of the attribute.
        attribute->Enabled = true;

        // Display the new value of the attribute's Enabled property.
        Console::WriteLine("ObjectPoolingAttribute.Enabled: {0}",
            attribute->Enabled);
    }
};